app/models/api/area.rb

```diff
@@ -7,6 +7,11 @@ class Scaling < ActiveResource::Base
     self.site = Api::Area.site
   end
 
+  class Init < ActiveResource::Base
```

Comment:
Why are the initializer inputs sent with the area?

They're an implementation detail, and since there is (currently?) no plan to add a front-end editor for them it seems unnecessary to include them with the response.